portfolio/src/main/webapp/scripts/controllers/comments_controller.js

```diff
@@ -67,4 +67,21 @@ slider.oninput = function() {
   renderCommentList(commentList.slice(0, this.value));
 };
 
-export { requestComments };
\ No newline at end of file
+const addNewCommentToList = newComment => {
+  commentList.unshift(newComment);
+  setSliderProps();
+  formatCommentHeaderText(slider.value, commentList.length);
+  renderCommentList(commentList.slice(0, slider.value));
+}
+
+const url = 'https://socket-dot-scarletnguyen-step-2020.uk.r.appspot.com/';
+const socket = io.connect(url);
+socket.on("comment-updates", newComment => {
+  if (newComment.message !== undefined && newComment.message === "deleted") {
+    requestComments('GET', null);
```

Comment:
Fine for now but think of how you could design this to avoid making this call when you already have N - 1 information.

Also -- there is a race condition:

User A sends DELETE
Server writes DELETE to DB

User A sends message X
Server writes message X to DB

Socket tells User B about DELETE
User B requests messages from DB
DB returns message X

Socket tells user about message X

Now message X is duplicated in User B's interface.